src/core/configmgr.cpp

```diff
@@ -378,9 +378,14 @@ QString ConfigMgr::getLogFile() const
 
 QString ConfigMgr::getApplicationFilePath()
 {
-#if defined(Q_OS_Linux)
+#if defined(Q_OS_LINUX)
     // TODO: Check if it is from AppImage.
     // We could get the APPIMAGE env variable from the AppRun script and pass it to vnote via cmd.
+    auto appImageVar = QString::fromLocal8Bit(qgetenv("APPIMAGE"));
+    qInfo() << "APPIMAGE" << appImageVar;
+    if (!appImageVar.isEmpty()) {
+        return appImageVar;
+    }
```

Comment:
I don’t think it’s necessary to decompress again. 
return QCoreApplication::applicationFilePath() is okay

Reply:
Yes, that is fine. I will use it only for the portable configuration folder.